bin/purge-habitat

```diff
@@ -15,46 +15,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+#
+# This script is based off of https://gist.github.com/cosimo/3760587
 
 set -eou pipefail
 
-INSTALL_TOOL_DIR="${INSTALL_TOOL_DIR:-$HOME/ci-studio-common/tools}"
-
 function usage() {
   cat <<DOC
-Usage: ${0##*/} TOOL [VERSION]
-
-Install one of the pre-configured TOOLs (listed below) at the specified VERSION (if specified).
+Usage: ${0##*/} [OPTIONS]
 
-AVAILABLE TOOLS:
-    chefdk [CHANNEL]
-    hab
+Uninstall Habitat from this machine.
 
-DEPRECATED TOOLS:
-    aws
-    docker-compose [VERSION]
-    terraform [VERSION]
+OPTIONS:
+  --all   Completely remove the Habitat installation.
 DOC
 }
 
-function install_tool() {
-  # Create the tools bin directory
-  mkdir -p "$HOME/tools/bin"
-
-  # Install the TOOL
-  tool="$1"
-  param="${2:-}"
-  tool_file="$INSTALL_TOOL_DIR/$tool/install.sh"
-
-  if [[ ! -f "$tool_file" ]]; then
-    usage
-    exit 1
-  else
-    ceval "bash $tool_file $param"
-  fi
+function uninstall_habitat() {
+  echo "--- Uninstalling Habitat"
+  hab_bin=$(command -v hab)
+  rm -rf /hab
+  rm -rf "$hab_bin"
```

Comment:
Since
```bash
  hab_bin=$(command -v hab)
```
will fail and cause the script to exit (per `-e`) without removing `/hab` if the `hab` command is missing or not in the path, it might make sense to change this to:
```bash
  if command -v hab &>/dev/null; then
    rm -f $(command -v hab)
  fi
```
It also makes the script idempotent, assuming you don't want to return an error if the install has already been removed.

Finally, there's no need for `-r`, as this is a file, not a directory.